### ISSUE TITLE
Touch screen control?

### ISSUE THREAD

**Issue by mrdROb** [pingumacpenguin/STM32-O-Scope#12]
Guys, have anyone made a touch screen controls for this wonderful project?

**Comment by Malmik** [comment]
Yes. https://www.instructables.com/id/Fast-Portable-and-Affordable-Oscilloscope-and-Indu
